uint32 image_width
uint32 image_height
uint32 min_x
uint32 max_x
uint32 min_y
uint32 max_y